examples/rapids/01-rapids-single-gpu.ipynb

```diff
@@ -0,0 +1,420 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use RAPIDS on a Single GPU"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![RAPIDS Logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/rapids.png)\n",
+    "\n",
+    "[RAPIDS](https://rapids.ai/) is a collection of libraries that enable you to take advantage of NVIDIA GPUs to accelerate machine learning workflows. Minimal changes are required to transition from familiar `pandas` and `scikit-learn` code!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "This example describes how to run a machine learning training workflow using the famous NYC Taxi Dataset. This dataset contains information about taxi trips in New York City. For the purposes of this example, we will be looking at the Yellow Taxi data from January 2019.\n",
+    "We will use this data to answer the following question:\n",
+    "> Based on characteristics that can be known at the beginning of a trip, will this trip result in a good tip?\n",
+    "\n",
```

Comment:
I would add a line at the beginning here saying something like "We are going to use RAPIDS to train a random forest model that takes features of the trips as inputs and tries to predict the boolean value of if the tip is >20% or not."

Just because otherwise we don't talk about what model we're using until like 2/3rds of the way through